### DISCUSSION TITLE
Add Mod

### DISCUSSION THREAD

**Discussion by Jaliosky** [Khopa/dcs_liberation#2924] [Q&A] [answered]
How to add the Codename flanker Su-30 mod

**Comment by Starfire13** [comment, marked as answer]
https://github.com/dcs-liberation/dcs_liberation/wiki/Modded-Aircraft-Support

**Reply by Jaliosky** [reply]
Well, it would be a good option to add this fantastic mod to release. Thanks for the attention

**Reply by Starfire13** [reply]
> Well, it would be a good option to add this fantastic mod to release. Thanks for the attention

You still haven't filed a mod support request though. Mod support is dependant on the community adding it in, which doesn't happen if no request is filed. You'll either have to follow the instructions on the wiki and implement it yourself, or file a request for it (not here in Q&A). :)
